### DISCUSSION TITLE
Caddy v2 Reverse Proxy stuck before login screen

### DISCUSSION THREAD

**Discussion by juliangaal** [filebrowser/filebrowser#1206] [Q&A]
**Description**
I am serving filebrowser behind a reverse proxy, with caddy v2. Adapted the caddy v1 tutorial on the website to my needs

baseurl: /filebrowser
url in caddy: /filebrowser

**Expected behaviour**
Login screen loads

**What is happening instead?**
I am stuck before the login screen at the three dots loading.

**Additional context**
Add any other context about the problem here. If applicable, add screenshots to help explain your problem.

**How to reproduce?**
My caddy config:
```
do.main {
  reverse_proxy /filebrowser http://127.0.0.1:8080
}
```

My filebrowser config:
```
{
  "port": 8080,
  "baseURL": "/filebrowser",
  "address": "127.0.0.1",
  "log": "stdout",
  "database": "/etc/filebrowser/database.db",
  "root": "/etc/filebrowser/files"
}

```

Thanks a lot! Your project looks awesome on my local machine

**Comment by WeidiDeng** [comment]
use

```
reverse_proxy /filebrowser* http://127.0.0.1:8080
```

The asterisk is essential, caddy 2 path match is exact by default
